extensions/amp-story/1.0/amp-story-tooltip.css

```diff
@@ -0,0 +1,152 @@
+  background-color: black !important;
+  opacity: 0.3 !important;
+  transition: opacity 0.15s cubic-bezier(0.4, 0.0, 1, 1) !important;
+}
+
+.i-amphtml-story-tooltip-layer.i-amphtml-hidden,
+.i-amphtml-story-tooltip-layer.i-amphtml-hidden::before {
+  opacity: 0 !important;
+  pointer-events: none !important;
+}
+
+.i-amphtml-story-tooltip-layer-nav-button-container {
+  height: 100% !important;
+  width: 160px !important;
+  position: absolute !important;
+}
+
+.i-amphtml-story-tooltip-layer-nav-button-container.i-amphtml-story-tooltip-nav-button-left {
+  background-image: linear-gradient(90deg, rgba(33,33,33,0.30) 2%, rgba(33,33,33,0) 100%) !important;
+  left: 0 !important;
+}
+
+.i-amphtml-story-tooltip-layer-nav-button-container.i-amphtml-story-tooltip-nav-button-right {
+  background-image: linear-gradient(90deg, rgba(33,33,33,0) 2%, rgba(33,33,33,0.30) 100%) !important;
+  right: 0 !important;
+}
+
+.i-amphtml-story-tooltip-layer-nav-button {
+  position: absolute !important;
+  top: 50% !important;
+  width: 48px !important;
+  height: 48px !important;
+  padding: 0 !important;
+  border: 0 !important;
+  background-color: transparent !important;
+  transition: transform 0.225s cubic-bezier(0.0, 0.0, 0.2, 1) !important;
+}
+
+.i-amphtml-story-tooltip-layer-nav-button.i-amphtml-story-tooltip-nav-button-left {
+  left: 0 !important;
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="48px" height="48px" viewBox="0 0 48 48" fill="#FFFFFF"><path d="M30.83 14.83L28 12 16 24l12 12 2.83-2.83L21.66 24z"/><path d="M0 0h48v48H0z" fill="none"/></svg>') !important;
+}
+
+.i-amphtml-hidden .i-amphtml-story-tooltip-layer-nav-button.i-amphtml-story-tooltip-nav-button-left {
+  transform: translate3d(8px, 0, 0);
+}
+
+.i-amphtml-story-tooltip-layer-nav-button.i-amphtml-story-tooltip-nav-button-right {
+  right: 0 !important;
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="48px" height="48px" viewBox="0 0 48 48" fill="#FFFFFF"><path d="M20 12l-2.83 2.83L26.34 24l-9.17 9.17L20 36l12-12z"/><path d="M0 0h48v48H0z" fill="none"/></svg>') !important;
+}
+
+.i-amphtml-hidden .i-amphtml-story-tooltip-layer-nav-button.i-amphtml-story-tooltip-nav-button-right {
+  transform: translate3d(-8px, 0, 0);
+}
+
+[desktop] .i-amphtml-story-tooltip-layer-nav-button-container {
+  display: none !important;
+}
+
+.i-amphtml-hidden > .i-amphtml-story-tooltip {
+  transform: translate3d(0, 8px, 0) !important;
+}
+
+.i-amphtml-story-tooltip {
+  max-width: 248px !important;
+  height: 40px !important;
+  border-radius: 6px !important;
+  padding: 8px !important;
+  position: absolute !important;
+  box-sizing: border-box !important;
+  text-decoration: none !important;
+  display: flex !important;
+  justify-content: space-between !important;
+  align-items: center !important;
+  background: white !important;
+  box-shadow: 0 4px 16px 0 rgba(0,0,0,0.24) !important;
```

Comment:
These numbers seem a bit extreme. It's hard to notice since it's displayed on top of an opacity layer anyway.
A more material-like box shadow would look like
`box-shadow: 0 1px 3px 1px rgba(0, 0, 0, 0.24)`